src/init.ts

```diff
@@ -49,16 +49,13 @@ async function generatePackageJson(
     packageJson: any, options: Options): Promise<void> {
   let edits = false;
   const outDir = 'build/';
-  const defaultSources = 'src/*.ts test/*.ts';
   const scripts: Bag<string> = {
     build: 'npm run compile',
     clean: 'rm -rf ./build/',
     compile: `tsc -p . --rootDir . --outDir ${outDir}`,
     // TODO(ofrobots): use a clang-format wrapper so that it can use the files,
     // include, and exclude sections from tsconfig.json.
-    format:
-        `clang-format -i -style="{Language: JavaScript, BasedOnStyle: Google,` +
-        ` ColumnLimit: 80}" ${defaultSources}`,
+    fix: `gts fix`,
```

Comment:
nit: Move the todo to `fix.ts`: we should use the `tsconfig.json`'s `files`, `includes` and `excludes` sections to figure out which files should be subject to fix.